packages/generator-superset/test/app.test.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const assert = require('yeoman-assert');
+const helpers = require('yeoman-test');
+
+describe('generator-superset:app', () => {
+  let dir;
+
+  beforeAll(() => {
+    dir = process.cwd();
```

Comment:
could we add a comment for why we `cd` since it was so tricky to figure this out?

Reply:
Yes!